tests/foreman/cli/test_errata.py

```diff
@@ -1091,22 +1092,28 @@ def test_positive_list_filter_by_product_id_and_org_label(self):
             set([])
         )
 
-    @stubbed()
-    def test_positive_list_filter_by_product_name(self):
```

Comment:
@oshtaier, for filtering by product_name and org  there is other positive tests implemented: 
test_positive_list_filter_by_product_name_and_org_id
test_positive_list_filter_by_product_name_and_org_name
test_positive_list_filter_by_product_name_and_org_label